Population/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Population
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //List<City> citiesList = new List<City>();
+            string listOfCities = "Kharkiv=1431000,Kiev=2804000,Las Vegas=603400";
+            string[] dividors = { "," };
+            string[] cities = listOfCities.Split(dividors, StringSplitOptions.RemoveEmptyEntries);
+            int maxCityNameLength = 0;
+            int maxCityPopulation = 0;
+            foreach(var city in cities)
+            {
+                string[] innerDividors = { "=" };
+                int dividorPosition = city.IndexOf(innerDividors[0]);
```

Comment:
You could just use 'Split' one more time here. Your approach is better in the case, if the value after split symbol can hold the split symbol itself.